src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -443,9 +486,40 @@ internal void InvokeResponse(InvocationResponse invokeResponse)
                 {
                     context.ResultSource.TrySetException(responseEx);
                 }
+                finally
+                {
+                    // Free memory allocated by the host (for input bindings)
+                    if (!_sharedMemoryManager.TryFreeSharedMemoryMapsForInvocation(invokeResponse.InvocationId))
+                    {
+                        _workerChannelLogger.LogWarning($"Cannot free all shared memory resources for invocation: {invokeResponse.InvocationId}");
```

Comment:
When would this ever happen? It seems if it was happening in a process, we wouldn't want to silently continue like this since it would mean we have a memory leak that will get worse over time?

Reply:
Only if `SharedMemoryManager::TryFreeSharedMemoryMap` returns `false` which happens if an attempt is made to remove an element from a dictionary that does not exist. A very unlikely scenario but kept this for sanity.